application.go

```diff
@@ -179,6 +179,29 @@ func (r *Application) Memory(memory float64) *Application {
 	return r
 }
 
+// AddPortDefinition adds a port definition. Port definitions are used to define ports that
+// should be considered part of a resource. They are necessary when you are using HOST
+// networking and no port mappings are specified.
+func (r *Application) AddPortDefinition(portDefinition PortDefinition) *Application {
+	if r.PortDefinitions == nil {
+		r.PortDefinitions = &[]PortDefinition{}
```

Comment:
@timoreimann ah yep! meant to change that after I added EmptyPortDefinitions in fact - will follow up!